Makefile

```diff
@@ -56,6 +56,11 @@ targets:
 # Build targets
 # -------------
 
+.PHONY: prepare
+prepare: ## Install dependencies
+	mix deps.get
+	npm ci --prefix assets
```

Comment:
Should we not separate elixir dependencies and node dependencies to be able to cache and execute step independently regarding if cache exists ?

We could have in `ci.yml` something like 
```
- if: steps.cache-mix.outputs.cache-hit != 'true'
   run: make dependencies
```

Reply:
I don’t think we need to check if there was a cache hit or not. In the end, after restoring the caches, `mix deps.get` will simply run is a few miliseconds!

As for `npm`, as mentionned above, the recommanded caching uses a different approach following `actions/cache@v1` documentation:

> For npm, cache files are stored in ~/.npm on Posix, or %AppData%/npm-cache on Windows. See https://docs.npmjs.com/cli/cache#cache
>
> _Note_: It is not recommended to cache `node_modules`, as it can break across Node versions and won't work with `npm ci`

https://github.com/actions/cache/blob/master/examples.md#node---npm